MapboxDirections/MBComponentRepresentable.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+/**
+The component representable protocol that comprises what the instruction banner should display.
+ */
+@objc(MBComponentRepresentable)
+public protocol MBComponentRepresentable: class, NSSecureCoding { }
```

Comment:
Type prefixes in swift are somewhat of an anti-pattern, unless your team has a special reason (and convention) for using them. They were [a convention](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingBasics.html#//apple_ref/doc/uid/20001281-1002226-BBCJECED) in Objective-C because all namespaces in that language are global, but with swift's introduction of [modules](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/AccessControl.html#//apple_ref/doc/uid/TP40014097-CH41-ID4), this pattern is more or less deprecated. This is because even if you call your class `UIViewController`, `MyApp.UIViewController()` is different from `UIKit.UIViewController()`.

 Naming the OBJ-C bridge `MBComponentRepresentable` is a good call though and should be kept.